macros/cross_db_utils/intersect.sql

```diff
@@ -0,0 +1,16 @@
+{% macro intersect() %}
```

Comment:
hmm, how did this one get in here?

Reply:
mmmh no idea, did this get added when I rebased? Says last updated 9 days ago by Claire, and I opened this PR before then?

Reply:
<img width="562" alt="image" src="https://user-images.githubusercontent.com/7342749/59947144-560c7780-9421-11e9-896f-4bee689e598d.png">

Reply:
yeah - i think that's right. I'd try rebasing against `master` again - that should make this commit disappear i think?

Reply:
Meh, I think this didn't do it. Now I have a lot more files from the rebase in this commit.
I fetched from upstream, merged upstream/master into my master, then rebased this branch from my local master. Does that sound right?

Reply:
hmm, yeah, that sounds like the right workflow to me! Not sure how you ended up with so many of these commits floating around. Lmk how I can help get it sorted - happy to chat on slack if that's easier :)